src/twisted/mail/test/test_imap.py

```diff
@@ -296,6 +296,86 @@ def cbProduced(result):
 
 
 
+class MessageSetTests(unittest.SynchronousTestCase):
+    """
```

Comment:
[This commit](https://codecov.io/gh/twisted/twisted/src/9f61890d97c41065dbb5cece2bd4af986c027363/src/twisted/mail/imap4.py) decreased [the previous commit's coverage](https://codecov.io/gh/twisted/twisted/src/562496b261a71b3372c7dd8015f556d32f1285cb/src/twisted/mail/imap4.py); [Two](https://codecov.io/gh/twisted/twisted/src/9f61890d97c41065dbb5cece2bd4af986c027363/src/twisted/mail/imap4.py#L279) [lines](https://codecov.io/gh/twisted/twisted/src/9f61890d97c41065dbb5cece2bd4af986c027363/src/twisted/mail/imap4.py#L282) aren't covered because this commit no longer passes the output of [`parsedIdList` to `MessageSet`](https://github.com/twisted/twisted/pull/817/commits/9f61890d97c41065dbb5cece2bd4af986c027363?diff=unified#diff-904151008804a75ba425f52b818531c8L399), which [doesn't make sense](https://twistedmatrix.com/trac/ticket/9178).